department-of-veterans-affairs/va.gov-cms
# VACMS-5864: Correct vc tooltip text.

## Description

Closes #5864

## Testing done
Visual / behat

## QA steps
 - As an admin, go to a vc form, and visually verify the `Featured content` and `How we're different` content tooltips have been updated.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
